scrapers_next/ma/committees.py

```diff
@@ -0,0 +1,100 @@
+from spatula import URL, HtmlListPage, HtmlPage, CSS, XPath, selectors
+from openstates.models import ScrapeCommittee
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+
+class CommitteeDetail(HtmlPage):
+    def process_page(self):
+        com = self.input
+        try:
+            for members_area in XPath(
+                '//*[contains(@class, "committeeMemberList")]'
+            ).match_one(self.root)[0]:
+                if "There are no members" in members_area.text:
+                    break
+                # continue to extract members if available
+                for members_info in XPath(
+                    '//*[contains(@class, "committeeMemberList")]'
+                ).match_one(members_area):
+                    for data in members_info:
+                        name = None
+                        role = "Member"
+                        for info in data.text_content().splitlines():
+                            if info.strip():
```

Comment:
As I mentioned elsewhere:
```
if not info:
    continue
```
Might be nicer than indenting the actual comparison another level.

Reply:
actually info some times has spaces only... so I'm checking if info.strip()